### PR TITLE
Add some benchmarks with unicode characters

### PR DESCRIPTION
**Pull request by Yaraslaut**
- [x]  single codepoint grapheme clusters (aka. US-ASCII) should always be the fastest.
- [x]  Diacritic
- [x]  Diacritic for two codepoints
- [x]  U+FE0F (VS16) for narrow-width GCs may affect perf due to width change (Diacritic)
- [ ]  invalid UTF-8 sequences might severily impact performance for 2 reasons (parsing as well as rendering that can impact parsing, depending on how a TE is implementing it)

### CODE REVIEW

File: scripts/Xvfb-bench-run.sh [hunk 1]
```diff
@@ -47,12 +49,12 @@ function program_exit() {
...
-    time "${@}" -e "${TB_BIN}" --fixed-size --stdout-fastpath --column-by-column --output "${OUTPUT_DIR}/${terminal_name}_results"
+    time "${@}" -e "${TB_BIN}" --fixed-size --column-by-column --size 2 --output "${OUTPUT_DIR}/${terminal_name}_results"
     local exit_code=$?
     printf "\033[1m==> Terminal exit code: $exit_code\033[m\n"
-    if [[ $exit_code -ne 0 ]]; then
-        program_exit $exit_code
-    fi
+    # if [[ $exit_code -ne 0 ]]; then
+    #     program_exit $exit_code
+    # fi
```
Comment (christianparpart) [inline scripts/Xvfb-bench-run.sh:57]: If we don't want that, I think we can simply drop it. What do you think?

File: tb/main.cpp [hunk 2]
```diff
@@ -148,7 +158,12 @@ BenchSettings parseArguments(int argc, char const* argv[], TerminalSize const& i
         else if (argv[i] == "--column-by-column"sv)
         {
             cout << std::format("Enabling column-by-column tests.\n");
-            settings.columnByColumn = true;
+            settings.tests.columnByColumn = true;
+            settings.tests.manyLines = false;
+            settings.tests.longLines = false;
+            settings.tests.sgrLines = false;
+            settings.tests.sgrFgBgLines = false;
+            settings.tests.binary = false;
```
Comment (christianparpart) [inline tb/main.cpp:166]: nice. this makes it slightly more explicit. :)

**Review by christianparpart — APPROVED** [review]
just a few comments

File: scripts/Xvfb-bench-run.sh (hunk 1 above)
Comment (Yaraslaut) [reply scripts/Xvfb-bench-run.sh]: > If we don't want that, I think we can simply drop it. What do you think?

deleted this part

**Comment by christianparpart** [comment]
Thank you:)

**Pull request merged**